README.md

```diff
@@ -33,7 +33,7 @@ The ChartIQ Angular application was built using the Angular 13 framework with [A
 
 ## Requirements
 
-A copy of the ChartIQ library, version 8.8.1 or later.
+A copy of the ChartIQ library, version 8.7.0 or later.
```

Comment:
Something from the publishing of the previous release didn't make it back into the private repo?

Reply:
Not sure, but it should be resolved in the private repo and then re-pubished, not modified here.